platform/mbed_stats.h

```diff
@@ -76,6 +95,16 @@ void mbed_stats_stack_get(mbed_stats_stack_t *stats);
  */
 size_t mbed_stats_stack_get_each(mbed_stats_stack_t *stats, size_t count);
 
+/**
+ *  Fill the passed array of stat structures with the thread stats for each available thread.
+ *
+ *  @param stats    A pointer to an array of mbed_stats_thread_t structures to fill
+ *  @param count    The number of mbed_stats_thread_t structures in the provided array
+ *  @return         The number of mbed_stats_thread_t structures that have been filled,
+ *                  this is equal to the number of stacks on the system.
```

Comment:
`this is equal to the number of stacks on the system` - I do not understand stacks in this  context? how many stacks we have? not threads here?